clap_complete/tests/testsuite/dynamic.rs

```diff
@@ -33,6 +33,37 @@ help\tPrint this message or the help of the given subcommand(s)"
     );
 }
 
+#[test]
+fn suggest_subcommand_aliases() {
```

Comment:
Please move this test commit to be before the `feat` commit, making it pass in each commit

Reply:
I'm not seeing this done though this was marked as resolved

Reply:
Sorry, I missed this.